Ball of Duty Server/Domain/Entities/Character.cs

```diff
@@ -79,5 +97,7 @@ public override bool Destroy(GameObject exterminator)
             }
             return false;
         }
+
+        public abstract void UpdateServerStats();
```

Comment:
pls name it UpdateStats. Character will never be responsible for updating anything clientside either way